pkg/bee/debugapi/node.go

```diff
@@ -249,3 +251,29 @@ func (n *NodeService) Topology(ctx context.Context) (resp Topology, err error) {
 
 	return
 }
+
+type Wallet struct {
+	BZZ         *bigint.BigInt `json:"bzzBalance"`
+	NativeToken *bigint.BigInt `json:"nativeTokenBalance"`
+}
+
+// Wallet returns the wallet state
+func (n *NodeService) Wallet(ctx context.Context) (resp Wallet, err error) {
+	err = n.client.requestJSON(ctx, http.MethodGet, "/wallet", nil, &resp)
+	return
+}
+
+// Withdraw calls wallet withdraw endpoint
+func (n *NodeService) Withdraw(ctx context.Context, token, addr string) (tx common.Hash, err error) {
+	endpoint := fmt.Sprintf("/wallet/withdraw/%s?address=%s&amount=1000000", token, addr)
```

Comment:
why hard code amount here?

Reply:
it's unlikely to change so didn't want to make it configurable

Reply:
moved it to a constant on the callers side